src/main/groovy/wooga/gradle/release/ReleasePlugin.groovy

```diff
@@ -131,13 +133,41 @@ class ReleasePlugin implements Plugin<Project> {
             //the release plugin sets this object as version to all projects
             project.version.toString()
             githubPublishTask.body(new ReleaseBodyStrategy(project.version.inferredVersion as ReleaseVersion, releaseExtension.grgit))
+
+            //create release note tasks
+            createReleaseNoteTasks(releaseExtension.grgit)
         }
 
         configureVersionCode(project)
         configureUnityPackageIfPresent(project)
         configurePaketConfigurationArtifacts(project)
     }
 
+    void createReleaseNoteTasks(Grgit git) {
+        GithubPublish githubPublishTask = (GithubPublish) tasks.getByName(GithubPublishPlugin.PUBLISH_TASK_NAME)
+
+        GenerateReleaseNotes appendLatestRelease = (GenerateReleaseNotes) tasks.create("appendReleaseNotes", GenerateReleaseNotes)
+        appendLatestRelease.appendLatestRelease(true)
+
+        def generateReleaseNotes = tasks.create("generateReleaseNotes", GenerateReleaseNotes)
+
+        [appendLatestRelease, generateReleaseNotes].each {
+            it.git(git)
+            it.releaseNotes(project.file("RELEASE_NOTES.md"))
```

Comment:
Yes because this whole plugin is not supposed to be configurable. It is a big opinionated configuration.
And this you can still reconfigure it in the project

**build.gradle**
```
generateReleaseNotes {
    releaseNotes file("Some other file")
}
```